content/applications/inventory_and_mrp/inventory/management/delivery/dropshipping.rst

```diff
@@ -1,109 +1,80 @@
+supplier is responsible for storing and shipping the item. This benefits the seller by reducing
+inventory costs, including the price of operating warehouses.
+
+Configure products to be dropshipped
+====================================
 
-Configure Drop-Shipping
-=======================
+To use dropshipping as a fulfillment strategy, navigate to the :guilabel:`Purchase` app and
+select :menuselection:`Configuration --> Settings`. Under the *Logistics* heading, click the
+:guilabel:`Dropshipping` checkbox.
 
-Activate the functionality in the *Purchase* application by going to :menuselection:`Configuration
---> Settings`.
+Next, go to the :guilabel:`Sales` app, click :menuselection:`Products --> Products` and choose an
+existing product or select :guilabel:`Create` to configure a new one. On the product page, make sure
+that the :guilabel:`Can be Sold` and :guilabel:`Can be Purchased` checkboxes are enabled.
 
-.. image:: dropshipping/dropshipping_01.png
+.. image:: dropshipping/sold_purchased_checkboxes.png
     :align: center
+    :alt: Enable the "Can be Sold" and "Can be Purchased" checkboxes on the product form.
 
-Then, go to the *Inventory* app, in :menuselection:`Configuration --> Settings` to
-activate the *Multi-Step Routes* feature. It will allow you to make
-the *Route* field appear on the sale order lines to specify you send a
-product via drop-shipping.
+Click on the :guilabel:`Purchase` tab and specify a vendor and the price that they sell the product
+for. Multiple vendors can be added, but the vendor at the top of the list will be the one
+automatically selected for purchase orders.
 
-.. image:: dropshipping/dropshipping_02.png
+.. image:: dropshipping/product_vendor_config.png
     :align: center
+    :alt: The product form with a vendor specified.
 
-Now, in the *Sales* app, go to :menuselection:`Products --> Products`. Select the
-product you would like to drop-ship and add a vendor pricelist which
-contains the right supplier, via the purchase tab.
+Finally, select the :guilabel:`Inventory` tab and enable the :guilabel:`Dropship` checkbox in the
+*Routes* section.
 
-.. image:: dropshipping/dropshipping_03.png
+.. image:: dropshipping/enable_dropship_route.png
     :align: center
+    :alt: Enable the Dropship option in the product inventory tab.
 
-Send Products from the Suppliers Directly to the Customers
-==========================================================
+Fulfill orders using dropshipping
+=================================
 
-Create a *Sales Order* and add the product on which you just set the
-vendor. Add the *Route* field thanks to the widget on the right of the
-sale order line. Now, you are able to specify that your products’ route
-is *Drop-Shipping*.
+When a customer completes an online purchase for a dropshipped product, a sales order will be
+automatically created with an associated request for quotation (RFQ) for the supplier. Sales orders
```

Comment:
Add the :abbr:`RFQ (Request for Quotation)` in the parenthesis and capitalize Request & Quotation